srcpkgs/cryptsetup/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'cryptsetup'
 pkgname=cryptsetup
-version=2.6.0
-revision=2
+version=2.6.1
+revision=1
 build_style=gnu-configure
-configure_args="--with-crypto_backend=openssl --enable-cryptsetup-reencrypt
```

Comment:
`cryptsetup-reencrypt` and this build option have been removed in https://gitlab.com/cryptsetup/cryptsetup/-/blob/5d622102c68fc7427d3a41b948be4fe596722f2f/docs/v2.5.0-ReleaseNotes#L29.